app-android-xamarin/Moodis/Moodis/Database/DatabaseModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Moodis.Feature.Login;
+
+namespace Moodis.Database
+{
+    class databaseModel
```

Comment:
Class names start with capital letters

Reply:
yeah, I know, but at first DatabaseModel was throwing naming errors and databaseModel was not, so I left it. After some tinkering I managed to fix it